CHANGELOG.md

```diff
@@ -448,9 +452,9 @@ Header Changes:
 
 #### Search
 
-* We removed the concepts seqan3::[bi_]fm_index[_cursor]_specialisation. We did this because we currently have only one
+* We removed the concepts `seqan3::[bi_]fm_index[_cursor]_specialisation`. We did this because we currently have only one
   implementation modelling each concept and are not completely sure if the current definition of the concepts is the
-  right one. If you used those concepts, you can check whether the cursor type is seqan3::[bi_]fm_index_cursor as a
+  right one. If you used those concepts, you can check whether the cursor type is `seqan3::[bi_]fm_index_cursor` as a
   substitute. ([\#2348](https://github.com/seqan/seqan3/pull/2348))
```

Comment:
FYI for reviewer: I noticed that the formatting is of here (`_` means italic in markdown), so i asked Mitra to fix this since she edits the file anyway.